shell/protoc.sh

```diff
@@ -2,7 +2,7 @@
 # This script is a light wrapper around the 'docker-protoc' image (see: https://github.com/namely/docker-protoc),
 # which is capable of compiling proto definitions into various languages.
 # This is not meant to be used in CircleCI (except by bootstrap itself to validate the general process).
-IMAGE="gcr.io/outreach-docker/protoc:latest"
+IMAGE="gcr.io/outreach-docker/protoc:1.37_2"
```

Comment:
the commit message is confusing `upgrade our grpc protoc tools to latest` 

where you are explicitly using the image number :(

Reply:
Yeah, this is technically a confusing commit message depending on the context. @masterkidan it might make more sense to change `latest` to `1.37_2` in the PR title